### ISSUE TITLE
The dataset seems to be unavailable

### ISSUE THREAD

**Issue by lijiandi** [globaldothealth/monkeypox#256]
Sorry to bother you. The latest data seems to be unavailable via the hyperlink. Any help would be appreciated. https://raw.githubusercontent.com/globaldothealth/monkeypox/main/latest.csv

**Comment by jim-sheldon** [comment]
Hello!

No need to apologize, we are grateful for your engagement and happy to help.
As indicated in the repository [README](https://github.com/globaldothealth/monkeypox#readme), we stopped including a `latest.csv` file in the repository in September.
We included a [link](https://7rydd2v2ra.execute-api.eu-central-1.amazonaws.com/web/) to our data files in the README; [here](https://7rydd2v2ra.execute-api.eu-central-1.amazonaws.com/web/url?folder=&file_name=latest.csv)'s the link to the last-updated `latest.csv` file.

I can download and view the file, so I will close this issue.
